designer/client/list/ListEdit.tsx

```diff
@@ -98,12 +98,20 @@ function useListEdit() {
   };
 }
 
+function validate(errors: any, selectedList: any) {
```

Comment:
Are we okay with anys in this project?

It's fine if we are I'm just checking.

Reply:
Yes i think so, it seems to be used throughout

Reply:
Okay in that case it's fine but going forward I think it would make sense to try to avoid using `any` and use the proper types. It might be more work but it's definitely worth it in the long run.